opennlp-tools/src/main/java/opennlp/tools/ml/naivebayes/Probabilities.java

```diff
@@ -146,14 +147,15 @@ public Double getLog(T t) {
       return normalised;
     Map<T, Double> temp = createMapDataStructure();
     double sum = 0;
-    for (T t : map.keySet()) {
-      Double p = map.get(t);
+    for (Entry<T, Double> entry : map.entrySet()) {
+      final Double p = entry.getValue();
```

Comment:
Not sure it makes sense to make it final. We did that only if necessary otherwise.